src/components/interfaces/MOBILE_API.xml

```diff
@@ -3060,16 +3060,20 @@
         <param name="hdRadioEnable" type="Boolean" mandatory="false"  since="5.0">
             <description> True if the hd radio is on, false if the radio is off</description>
         </param>
-        <param name="availableHDs" type="Integer" minvalue="1" maxvalue="7" mandatory="false" since="5.0">
-            <description>number of HD sub-channels if available</description>
+        <param name="availableHDs" type="Integer" minvalue="0" maxvalue="7" mandatory="false" deprecated="true" since="5.1">
```

Comment:
@IGapchuk is that mean deprecated since 5.1? Or will be deprecated before 5.1?

Reply:
@AKalinich-Luxoft Will be deprecated since 5.1. Fixed in 39d969d46ff3099f811e17443222a2af87425a20.